### PR TITLE
Russian translation of "Refuge / Wallet Quickstart"

### PR DESCRIPTION
**Pull request by v1docq47**
_add:_
>  monero-outreach-docs/translations/ru/storyboard/refuge_ru.md
>  monero-outreach-docs/translations/ru/storyboard/wallet_quickstart_ru.md

_translated:_
>  monero-outreach-docs/translations/ru/storyboard/refuge_ru.md
>  monero-outreach-docs/translations/ru/storyboard/wallet_quickstart_ru.md

Calling to review:
@monerorus @LvMsterfild @TheFuzzStone

### CODE REVIEW

**Comment by TheFuzzStone** [comment]
Done

File: monero-outreach-docs/translations/ru/storyboard/wallet_quickstart_ru.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+
+Программное обеспечение кошелька Monero представляет собой программу для хранения, отправки и получения Monero. Оно обеспечивает полный контроль над вашими Monero, гарантируя анонимность и отсутствие каких-либо посредников. Никто не сможет узнать всех подробностей ваших транзакций, равно как и не сможет изменить их, если только вы не поделитесь своими ключами просмотра для проведения аудита. Эта особенность является сильной стороной и сущностью Monero.
+
+Для того, чтобы использовать эту возможность, вам будет необходимо создать кошелёк. Первым критическим аспектом любого кошелька является его надёжность. В этой статье упоминаются только те кошельки, которые рекомендуются сообществом (и веб-сайтом [getmonero.org](https://getmonero.org/)). Затем, вам будет необходимо выбрать тот кошелёк, который будет работать с вашим вычислительным устройством: iPhone, мобильным телефоном с операционной системой Android или же с портативным или стационарным компьютером. Наконец, вы выберите тот кошелёк, который будет соответствовать вашим требованиям к уровню анонимности. И эта статья поможет вам сделать выбор.
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/storyboard/wallet_quickstart_ru.md]: For the link we can redirect to the russian getmonero site:

https://getmonero.org/ru/

File: monero-outreach-docs/translations/ru/storyboard/wallet_quickstart_ru.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
... если вы хотите запустить собственный полный узел, то хорошим выбором станет официальный GUI или CLI-кошелёк Monero. Первый кошелёк поддерживает графический интерфейс пользователя, а второй имеет текстовый интерфейс. Оба кошелька можно конфигурировать так, что они будут поддерживать либо собственный полный узел, либо удалённый узел с повышением уровня анонимности. Эти кошельки можно загрузить по ссылке: [getmonero.org/downloads](https://getmonero.org/downloads/). С устройствами, работающими под операционной системой Android или iOS, рекомендуется использовать лёгкие кошельки с защитой ключа просмотра: Monerujo для Android (можно скачать на [monerujo.io](https://www.monerujo.io/) и Cake Wallet для iOS (можно скачать на [cakewallet.io](https://cakewallet.io/)). Если же вы хотите создать кошелёк с возможностью совместного использования ключа просмотра, то вам рекомендуется установить браузерный кошелёк MyMonero (доступен для скачивания на [mymonero.com](https://mymonero.com/)). Edge Wallet (который можно скачать на [edge.app](https://edge.app/)) является другим солидным лёгким кошельком с возможностью совместного использования ключа просмотра, доступным как для Android, так и для iOS.
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/storyboard/wallet_quickstart_ru.md]: Will just post the missing links :):

https://web.getmonero.org/ru/downloads/

File: monero-outreach-docs/translations/ru/storyboard/wallet_quickstart_ru.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+Выбирая из этих кошельков, следует помнить о том, что все они (за исключением Edge Wallet) могут быть конфигурированы как для подсоединения к удалённому серверу третьей стороны, так и к вашему серверу, на котором работает ваш узел. Это проиллюстрировано в таблице.
+
+## Официальные кошельки
+
++ [getmonero.org/downloads](https://www.getmonero.org/downloads)
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/storyboard/wallet_quickstart_ru.md]: https://web.getmonero.org/ru/downloads/

File: monero-outreach-docs/translations/ru/storyboard/wallet_quickstart_ru.md (hunk 1 above)
Comment (v1docq47) [reply monero-outreach-docs/translations/ru/storyboard/wallet_quickstart_ru.md]: fix

File: monero-outreach-docs/translations/ru/storyboard/wallet_quickstart_ru.md (hunk 2 above)
Comment (v1docq47) [reply monero-outreach-docs/translations/ru/storyboard/wallet_quickstart_ru.md]: fix

File: monero-outreach-docs/translations/ru/storyboard/wallet_quickstart_ru.md (hunk 3 above)
Comment (v1docq47) [reply monero-outreach-docs/translations/ru/storyboard/wallet_quickstart_ru.md]: fix

**Comment by v1docq47** [comment]
@lh1008 thanks!
I fixed links ;)

**Pull request merged**